README.md

```diff
@@ -1,101 +1,117 @@
-**CircleCI has a new [container runner](https://circleci.com/docs/container-runner) in open preview, which will replace the self-hosted runner on Kubernetes solution detailed on this page.**
+**CircleCI has a new [Container runner](https://circleci.com/docs/container-runner) in open preview.**
 
-# CircleCI Runner on Kubernetes
+# Introduction
+This installation guide is to help set up self-hosted runners that use the Machine executor on your Kubernetes cluster. This is the deprecated way of using self-hosted runners with Kubernetes.  The docs are being kept here for users still using the deprecated method.  [Container runner](https://circleci.com/docs/container-runner) is the supported way for running self-hosted runners on Kubernetes.
```

Comment:
Since folks coming here may be legacy users, do we want to say "the Machine executor formerly known as Launch Agent" ?

Reply:
i think let's just say "launch agent". soon we'll have the docs changed to say "machine runner" and "container runner" and then we can change this to say "machine runner" lemme update it